library.properties

```
@@ -1,9 +1,10 @@
 name=ConfigTool
-version=1.0
-author=Tvde1
-maintainer=Tvde1
+version=1.1
```

```suggestion
version=2.0
```
It's quite a big change. Maybe use 2.0

---

I also noticed the version is `1.1` already. If `2.0` doesn't sound good then go with `1.2`.